### DISCUSSION TITLE
SRAM Quarq Powermeter with Osmand+ possible

### DISCUSSION THREAD

**Discussion by Schnubu** [osmandapp/OsmAnd#20706] [General]
Hi,
is it possible to connect OsmAnd with a SRAM Quarq powermeter via external sensors in OsmAnd?
https://www.sram.com/de/sram/models/pm-frc-kit-d2
I tried several times, bit OsmAnd will not find the device, even though the powermeter is connected to the Smartphone. I would really like to use OsmAnd instead of using another app. 
Do you have any tricks, to connect the Quarq powermeter?
Thanks in advance
Schnubu

**Comment by dmpr0** [comment]
Hi! Please try to connect it one more time, when it doesn’t work send us log files from Menu – Help – Help improve OsmAnd – Send logcat log.

**Reply by Schnubu** [reply]
Hi, connecting powermeter to other apps like IpBike or Locus Maps works fine. Connection to Osmand still not running. I tried it several times, two logcat-files are send out to you via mail.
I am looking forward using Osmand+ with powermeter. Thanks for your support.

**Reply by Schnubu** [reply]
Hi, any news?

**Reply by dmpr0** [reply]
We are working on adding additional logging for BLE sensors, when it will be done we will ask you to perform Pairing again.

**Comment by Schnubu** [comment]
Nice, thanks in advance.

 

Von: Dmytro Prodchenko ***@***.***> 
Gesendet: Montag, 2. September 2024 10:52
An: osmandapp/OsmAnd ***@***.***>
Cc: Schnubu ***@***.***>; Author ***@***.***>
Betreff: Re: [osmandapp/OsmAnd] SRAM Quarq Powermeter with Osmand+ possible (Discussion #20706)

 

We are working on adding additional logging for BLE sensors, when it will be done we will ask you to perform Pairing again.
